packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -87,10 +88,7 @@
 }
 
 .auto-crud-dialog {
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
+  --vaadin-overlay-viewport-inset: 0;
```

Comment:
The actual overlay element is now in the shadow root of the dialog and can not be targeted with global CSS, there isn't a part name for it either. To customize the offset between the viewport and the overlay this property needs to be used now. Otherwise it looked like this:

<img width="572" height="1041" alt="Bildschirmfoto 2025-09-04 um 10 34 42" src="https://github.com/user-attachments/assets/703b4cc8-1fe0-4324-96df-f35753823ec9" />